src/ducks/categorization/services.js

```diff
@@ -206,31 +209,57 @@ export const localModel = async (classifierOptions, transactions) => {
     { ...classifierOptions, ...options.initialization },
     options.configuration
   )
+  return classifier
+}
+
+export const pctOfTokensInVoc = (tokens, vocabularyArray) => {
+  const n_tokens = tokens.length
+  const intersection = tokens.filter(t => -1 !== vocabularyArray.indexOf(t))
+  return intersection.length / n_tokens
+}
+
+export const localModel = async (classifierOptions, transactions) => {
+  const classifier = await createLocalModel(classifierOptions)
 
-  if (!classifier) {
+  if (classifier !== undefined) {
```

Comment:
```suggestion
  if (classifier) {
```

Actually `createLocalClassifier` returns `null` if there is no manually categorized transactions. So checking only `undefined` is not enough and will try to categorize transactions with no model at all, crashing the process.

Sorry if I was not clear in my previous comment.